compiler/cpp/src/interpreter/Interpreter.cpp

```diff
@@ -52,7 +52,11 @@ std::string Interpreter::getOutputOfContent(ContentExprAST* content) {
 }
 
 std::string Interpreter::getOutputOfArbitrarySection(ArbitraryExprAST* arbitraryExpr) {
-    return arbitraryExpr->getValue();
+    std::string arbitrary = arbitraryExpr->getValue();
+    // Cut of first char, if it is a line break
+    if (arbitrary.rfind('\n', 0) == 0)
+        arbitrary.erase(0, 1);
+    return arbitrary;
```

Comment:
what's the purpose of modifying the arbitrary section here?

(also grammar: use `Cut out`)

Reply:
The problem was that we had unwanted line breaks in the compile output. This line breaks were caused by arbitrary sections that only contain `\n`. This code snippet fixes that issue.

Reply:
ah ok 👌

and yeah, `cut off` works as well ;)